src/lib/controllers/projectsRest/projectsRestController.ts

```diff
@@ -0,0 +1,54 @@
+import type { Controller, ProjectData, ProjectsResponse } from '../../types';
+import type { NextFunction, Request, Response } from 'express';
+import { Router } from 'express';
+import { getProjectsFromRestApi } from './projectsRestHandlers';
+
+/**
+ * The ProjectsRestController class for handling user projects
+ * page and related requests. Every controller class
+ * implements the controller interface which
+ * has two members the path and the router.
+ */
+export class ProjectsRestController implements Controller {
+  // The base URL path for this controller
+  public path = '/projects-rest';
+  // Express router for this controller
+  public router = Router();
+
+  /**
+   * The constructor is used to initialize the
+   * routes for this controller
+   */
+  constructor() {
+    this.initRoutes();
+  }
+
+  private initRoutes() {
+    // The route to render all user projects lists
+    this.router.get(`${this.path}`, this.getProjectsRest);
+  }
+
+  /**
+   * Gets the projects page from the Snyk REST API using the
+   * user access token and then renders the project list
```

Comment:
Nit: We probably need to improve the lingo a couple of place for this existing code as well 😅 
```suggestion
   * app's access token and then renders the project list
```